subsys/zigbee/cli/zigbee_cli_utils.c

```diff
@@ -140,6 +143,49 @@ int zb_cli_zcl_attr_to_str(char *str_buf, uint16_t buf_len,
 	return bytes_written;
 }
 
+zb_bool_t zb_cli_is_zcl_cmd_response(zb_zcl_parsed_hdr_t *zcl_hdr, struct ctx_entry *entry)
+{
+	zb_uint16_t remote_node_short = 0;
+	struct zcl_packet_info *packet_info = &entry->zcl_data.pkt_info;
+
+	if (packet_info->dst_addr_mode == ZB_APS_ADDR_MODE_64_ENDP_PRESENT) {
```

Comment:
I see some potential to cleanup the code even more:

```suggestion
	if (zcl_hdr->profile_id != packet_info->prof_id) {
		return ZB_FALSE;
	}

	if (zcl_hdr->addr_data.common_data.src_endpoint != packet_info->dst_ep) {
		return ZB_FALSE;
	}
	if (zcl_hdr->addr_data.common_data.source.addr_type != ZB_ZCL_ADDR_TYPE_SHORT) {
		return ZB_FALSE;
	}
	
	if (packet_info->dst_addr_mode == ZB_APS_ADDR_MODE_64_ENDP_PRESENT) {
		remote_node_short = zb_address_short_by_ieee(packet_info->dst_addr.addr_long);
	} else {
		remote_node_short = packet_info->dst_addr.addr_short;
	}
	if (zcl_hdr->addr_data.common_data.source.u.short_addr != remote_node_short) {
		return ZB_FALSE;
	}
	return ZB_TRUE;
```